pallets/capacity/src/lib.rs

```diff
@@ -100,19 +101,35 @@ pub mod pallet {
 	};
 	use sp_runtime::traits::{AtLeast32BitUnsigned, MaybeDisplay};
 
+	/// A reason for freezing funds.
+	/// Creates a freeze reason for this pallet that is aggregated by `construct_runtime`.
+	#[pallet::composite_enum]
+	pub enum FreezeReason {
+		/// The account has staked tokens to the Frequency network.
+		Staked,
+	}
+
+	/// the storage version for the v2 migration
+	pub const STORAGE_VERSION_V2: StorageVersion = StorageVersion::new(2);
```

Comment:
why do we need two of these?

Reply:
Capacity has to run 2 migrations in series. We could just write `2`, I thought `STORAGE_VERSION_V2` would be useful in more places.

Reply:
`STORAGE_VERSION_V2` has been moved to v2.rs to avoid confusion.